ozark/src/main/java/org/glassfish/ozark/binding/BindingResultImpl.java

```diff
@@ -89,22 +91,26 @@ public BindingError getBindingError(String param) {
     }
 
     @Override
-    public Set<ConstraintViolation<?>> getAllViolations() {
-        return violations;
+    public Set<ValidationError> getAllValidationErrors() {
+        return Collections.unmodifiableSet(validationErrors);
     }
 
     @Override
-    public Set<ConstraintViolation<?>> getViolations(String propertyPath) {
```

Comment:
I checked this again. Looks like it was called "propertyPath" only in the old version of the file. My patch seems to be fine and is using "param" instead.